### PR TITLE
Added Pull Request Template

### LINKED ISSUE #36: chore: add pull request template
# What

I'd like to make this repo easier to manage by creating a template for pull requests.

The documentation for creating templates on GitHub can be found here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository.

# What to include in the template

Here is what I would like to find in the template:

* a description section
* does this solve an issue? If so, reference the issue, or describe it
* some checkboxes to make stating
  * [ ] I understand that my contributions needs to pass the checks
  * [ ] If I create new functions / methods, I documented them and add type hints
  * [ ] If I modified already existing code, I updated the documentation accordingly
* additional contents: images, error output, ...

### PR DESCRIPTION
**Pull request by ahmedsilat44**
Closes #36

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<!-- Example: Closes #31 -->
+
+## Description
+
+<!-- Describe all the proposed changes in your PR -->
+
+## Check List (Check all the applicable boxes)
+
+- [ ] I understand that my contributions needs to pass the checks.
+- [ ] If I create new functions / methods, I documented them and add type hints.
```
Comment (jeertmans) [inline .github/pull_request_template.md:16]:
```suggestion
- [ ] If I created new functions / methods, I documented them and add type hints.
```

**Comment by jeertmans** [comment]
Hi @ahmedsilat44, thank you for your work!

I just suggested a small change, a small English mistake.

You can apply suggested changes, and I will merge this :-)

**Comment by ahmedsilat44** [comment]
> Hi @ahmedsilat44, thank you for your work!
> 
> 
> 
> I just suggested a small change, a small English mistake.
> 
> 
> 
> You can apply suggested changes, and I will merge this :-)

Done 👍

**Comment by jeertmans** [comment]
Thanks for your reactivity! Merged!

**Pull request merged**